api-flask/app/caching.py

```diff
@@ -21,7 +21,7 @@ def cache_file(url, prefix):
     )
     # If the file exists, return path.
     if os.path.isfile(cache_filepath):
-        logger.warning('Returning cached file for {}.'.format(url))
+        logger.info('Returning cached file for {}.'.format(url))
```

Comment:
`rasterio` is being VERY VERY noisy in debug, so that's why I decided to up the level of this to make them useful. The better alternative would be to disable debug logging for rasterio but I tried a few things without luck...